### DISCUSSION TITLE
Error installing speedtest tracker

### DISCUSSION THREAD

**Discussion by rollingafull** [runtipi/runtipi-appstore#2630] [Q&A]
Hi i've this error installing speedtest tracker, any idea how can i solve it?

`[root@tipi runtipi]# cat logs/error.log
worker: 2024-03-01T10:17:18.163Z - error >      An error occurred: Command failed: docker-compose --env-file /storage/app-data/speedtest-tracker/app.env --project-name speedtest-tracker -f /app/apps/speedtest-tracker/docker-compose.yml -f /app/repos/29ca930bfdaffa1dfabf5726336380ede7066bc53297e3c0c868b27c97282903/apps/docker-compose.common.yml up --detach --force-recreate --remove-orphans --pull always
 speedtest-tracker-db Pulling 
 speedtest-tracker Pulling 
 speedtest-tracker Pulled 
 speedtest-tracker-db Pulled 
 Container speedtest-tracker-db  Recreate
 Container speedtest-tracker-db  Recreated
 Container speedtest-tracker  Recreate
 Container speedtest-tracker  Recreated
 Container speedtest-tracker-db  Starting
 Container speedtest-tracker-db  Started
 Container speedtest-tracker  Starting
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/etc/localtime" to rootfs at "/etc/localtime": mount /etc/localtime:/etc/localtime (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

2024-03-01T10:17:18.169Z > Failed to install app speedtest-tracker: Command failed: docker-compose --env-file /storage/app-data/speedtest-tracker/app.env --project-name speedtest-tracker -f /app/apps/speedtest-tracker/docker-compose.yml -f /app/repos/29ca930bfdaffa1dfabf5726336380ede7066bc53297e3c0c868b27c97282903/apps/docker-compose.common.yml up --detach --force-recreate --remove-orphans --pull always
 speedtest-tracker-db Pulling 
 speedtest-tracker Pulling 
 speedtest-tracker Pulled 
 speedtest-tracker-db Pulled 
 Container speedtest-tracker-db  Recreate
 Container speedtest-tracker-db  Recreated
 Container speedtest-tracker  Recreate
 Container speedtest-tracker  Recreated
 Container speedtest-tracker-db  Starting
 Container speedtest-tracker-db  Started
 Container speedtest-tracker  Starting
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/etc/localtime" to rootfs at "/etc/localtime": mount /etc/localtime:/etc/localtime (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type`

`[root@tipi runtipi]# df -h
Filesystem                    Size  Used Avail Use% Mounted on
rpool/data/subvol-120-disk-0   64G  2.8G   62G   5% /
none                          492K  4.0K  488K   1% /dev
udev                           63G     0   63G   0% /dev/fuse
tmpfs                          63G     0   63G   0% /dev/shm
tmpfs                          26G  9.1M   26G   1% /run
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/18e97d316edc26096eca48ddce80561f1750635feaacb128ded885f74442329a/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/ed92609a26617c8def8ae7b9c73168ffdc7c8c17041fce5b920f2715b09d853d/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/780a50468be6afab663f70063c5923ccbbb8f6c3658f0fda1e74fa8ea0ce01af/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/0c2c86ca7521f4fba16bd185480991f287c98293b8903f0326927447e3dc3b59/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/db4f1caae2f8e92286449b89309edc392c7373e1087d7ec695a7cc330ccfdbfa/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/42790d12d9814ca6d67bbb26380af2a1643d1cf39154113cb76801fc46dc29a9/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/87f427d5d4f52ec99968c9d20d47edc1d19025b0ae10508168d75a462228ef74/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/91174858bb109ac5087399bee2d05769343040fc27cd3549d2abbcf7746a5de6/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/476ed5f0bdbe748420a89a6b4edef23d06e20a7efd9b6b3deda9dd3d3780daf7/merged
fuse-overlayfs                 64G  2.8G   62G   5% /var/lib/docker/fuse-overlayfs/bd66839d8b3f3cd49ec4d2b8b150114b853f67da6e4fca3677e598db9126eb34/merged
tmpfs                          13G     0   13G   0% /run/user/0`

`[root@tipi runtipi]# ls -la /etc/localtime/
total 9
drwxr-xr-x  2 root root   2 Feb 29 21:13 .
drwxr-xr-x 68 root root 148 Mar  1 10:12 ..`

Thanks.

**Comment by rollingafull** [comment]
Is realy necesary to mount this volume /etc/localtime?
I'm running Tipi into a LXC CT on Proxmox.

volumes:
  - ${APP_DATA_DIR}/data/speedtest-tracker/config:/config
  - /etc/localtime:/etc/localtime:ro

Regards.

**Reply by nicotsx** [reply]
No, I think we can remove this mount

**Comment by nicotsx** [comment]
I pushed an update removing the `/etc/localtime` volume
